client/controllers/articles.coffee

```diff
@@ -48,12 +48,12 @@ Template.articles.helpers
   selectedSource: ->
     selectedId = Template.instance().selectedSourceId.get()
     if selectedId
-      Articles.findOne selectedId
+      grid.Articles.findOne selectedId
```

Comment:
It would be better not to use the grid object. We have a plans to get rid of it: https://www.pivotaltracker.com/story/show/129994355